src/view.c

```diff
@@ -611,6 +612,41 @@ view_get_edge_snap_box(struct view *view, struct output *output,
 	return dst;
 }
 
+/* Move and resize view based on percent output local values in rel_box */
+void
+view_rel_move_resize_to(struct view *view, struct wlr_box *rel_box)
+{
+	assert(view);
+	assert(rel_box);
+
+	struct output *output = view_output(view);
+	if (!output) {
+		return;
+	}
+	struct wlr_box usable = output_usable_area_in_layout_coords(output);
+
+	/* Calculate new size and position */
+	struct wlr_box new_geo = { view->x, view->y, view->w, view->h };
+	if (rel_box->x != INT_MIN) {
```

Comment:
openbox spec doesn't allow for `<x>` and `<y>` expressed as percentages. Is it actually useful to do that??

Reply:
I think it is an absolute requirement for our use-case. How else would you have some bindings that allow to freely tile independent of any knowledge of the current output or its dimensions?
With this a user could easily split any output into a `2x2`, `3x3` tiling state or instead do something like this:
```
.----------.--------------------.----------.
|          |                    |          |
|          |                    |          |
|          |                    |----------|
|          |                    |          |
|          |                    |          |
|          |                    |----------|
|          |                    |          |
|          |                    |          |
`----------^--------------------^----------´
```